packages/bs-neo3/src/BSNeo3.ts

```diff
@@ -46,6 +50,59 @@ export class BSNeo3<BSCustomName extends string = string>
     this.feeToken = this.tokens.find(token => token.symbol === 'GAS')!
     this.tokenClaim = this.tokens.find(token => token.symbol === 'GAS')!
     this.setNetwork(network)
+    this.ghostMarket = new GhostMarket(this)
+  }
+  async getNFTS(address: string, page: number = 1): Promise<NFTSResponse> {
+    const nftPageLimit = 18
+    const { assets, total } = await this.ghostMarket.getNFTS({
+      owners: [address],
+      size: nftPageLimit,
+      page,
+      getTotal: true,
+    })
+
+    const totalPages = Math.ceil(total / nftPageLimit)
+    const nfts = assets.map<NFTResponse>(asset => ({
+      collectionImage: this.ghostMarket.treatGhostMarketImage(asset.collection.logoUrl),
+      collectionName: asset.collection.name,
+      image: this.ghostMarket.treatGhostMarketImage(asset.metadata.mediaUri),
+      name: asset.metadata.name,
+      symbol: asset.contract.symbol,
+      id: asset.tokenId,
+      contractHash: asset.contract.hash,
+      isSVG: String(asset.metadata.mediaType).includes('svg+xml'),
+    }))
+
+    const nftsResponse: NFTSResponse = {
+      totalPages: totalPages,
+      items: nfts
+    }
+
+    return nftsResponse
```

Comment:
It can be inside the getNFTS from GhostMarket class, there is no need to separate the query from the data treatment. If the same logic could be used for all nftProviders, it would make sense, but since each provider returns in a different way, it doesn't make sense to separate it, since we'll have to change both places anyway. So it's better to have everything

Reply:
was added 2 methods in the class to transform the response ghostmarket to NFT Response and NFTS Response, the methods get are returning the correct format now